internal/cfgplugins/policyforwarding.go

```diff
@@ -516,6 +528,31 @@ func DecapGroupConfigGue(t *testing.T, dut *ondatra.DUTDevice, pf *oc.NetworkIns
 	}
 }
 
+func GueDecapConfig(t *testing.T, dut *ondatra.DUTDevice, params OcPolicyForwardingParams) {
```

Comment:
This is arista specific, so should be something like:
```suggestion
func aristaGueDecapCLIConfig(t *testing.T, dut *ondatra.DUTDevice, params OcPolicyForwardingParams) {
```

Reply:
Addressed this comment